src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx

```diff
@@ -54,21 +54,35 @@ const NetworkInterfaceNetworkSelect: FC<NetworkInterfaceNetworkSelectProps> = ({
   const podNetworkingText = useMemo(() => t('Pod Networking'), [t]);
 
   const networkOptions = useMemo(() => {
-    const options = nads?.map(({ metadata }) => ({
-      key: metadata?.uid,
-      value: `${metadata?.namespace}/${metadata?.name}`,
-    }));
+    const options = nads?.map((nad) => {
+      const { name, namespace: nadNamespace, uid } = nad?.metadata;
+      return {
+        key: uid,
+        type: getNadType(nad),
+        value: `${nadNamespace}/${name}`,
+      };
+    });
     if (isPodNetworkingOptionExists) {
-      options.unshift({ key: 'pod-networking', value: podNetworkingText });
+      options.unshift({
+        key: 'pod-networking',
+        type: interfacesTypes.bridge,
+        value: podNetworkingText,
+      });
     }
     return options;
   }, [isPodNetworkingOptionExists, nads, podNetworkingText]);
 
+  useEffect(() => {
+    loaded && setInterfaceType(getNadType(nads[0]));
+  }, [loaded, nads, setInterfaceType]);
```

Comment:
is it guaranteed to have at least one nad? maybe add ? to avoid crashing here

Reply:
yescan add nothing wrong with that